client/src/components/EditMyAvailability.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from "react";
+import { Form, Modal } from "react-bootstrap";
+import Button from "react-bootstrap/Button";
+function EditMyAvailability(props) {
+  const [edit, setEdit] = useState(false);
+  const [trainees_id, setTrainees_id] = useState(localStorage.getItem("id"));
+  const [date, setDate] = React.useState(new Date());
+  const [topic, setTopic] = React.useState("");
+  const handleUpdate = (e) => {
```

Comment:
also here, an empty line below the hooks for redability

Reply:
Thank you, I have just fixed it.